org-journal.el

```diff
@@ -434,21 +443,22 @@ This runs once per date, before `org-journal-after-entry-create-hook'.")
                       #'concat
                     (lambda (prefix key) (kbd (concat prefix "" key))))))
     (cl-loop for (key . command) in command-table
-      do (define-key org-journal-mode-map (funcall key-func org-journal-prefix-key key) command))))
-
-(eval-after-load "calendar"
-  '(progn
-    (define-key calendar-mode-map (kbd "j m") 'org-journal-mark-entries)
-    (define-key calendar-mode-map (kbd "j r") 'org-journal-read-entry)
-    (define-key calendar-mode-map (kbd "j d") 'org-journal-display-entry)
-    (define-key calendar-mode-map "]" 'org-journal-next-entry)
-    (define-key calendar-mode-map "[" 'org-journal-previous-entry)
-    (define-key calendar-mode-map (kbd "j n") 'org-journal-new-date-entry)
-    (define-key calendar-mode-map (kbd "j s f") 'org-journal-search-forever)
-    (define-key calendar-mode-map (kbd "j s F") 'org-journal-search-future)
-    (define-key calendar-mode-map (kbd "j s w") 'org-journal-search-calendar-week)
-    (define-key calendar-mode-map (kbd "j s m") 'org-journal-search-calendar-month)
-    (define-key calendar-mode-map (kbd "j s y") 'org-journal-search-calendar-year)))
+             do (define-key org-journal-mode-map (funcall key-func org-journal-prefix-key key) command))))
+
+(if org-journal-enable-calendar-integration
```

Comment:
Would it break anything if these remained enabled? Or perhaps it would make sense to only disable the first five. I could see a use case for `org-journal-new-date-entry` and the `org-journal-search-*` functions even with the calendar highlights disabled.

Reply:
Good point.